website/static/images/destination1.svg

```diff
@@ -1,3 +1,19 @@
+<!--Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.-->
```

Comment:
In various cases this kind of icons come from commercial templates that are already under some kind of LICENSE. Depending on the LICENSE it may not be appropriate to re-license them under AL2.

If you created them yourself then its fine to add the header otherwise we should leave them as is and open a separate ticket to see if following actions are needed. In fact, we may need to open a separate ticket anyways to review all image related content.

Reply:
I went to traceback the history on this image and actually this image and a few others aren't actively used on the site. I will delete the unused images and actually all current ones are original hand-drawings.

Reply:
That's great news. It would greatly simplify things if we get rid of unused material.

Reply:
Deleted unused imgs in latest commit